src/organisms/SideBar/CreateItem.jsdom.test.tsx

```diff
@@ -0,0 +1,125 @@
+import { add } from '@equinor/eds-icons';
+import { faker } from '@faker-js/faker';
+
+import { colors, shape } from 'src/atoms/style';
+import { CreateItem, CreateItemProps } from 'src/organisms/SideBar/CreateItem';
+import { SideBarProvider } from 'src/providers/SideBarProvider';
+import { render, screen } from 'src/tests/jsdomtest-utils';
+
+function fakeProps(): CreateItemProps {
+  return {
+    createLabel: faker.company.buzzVerb(),
+    disabled: false,
+    onCreate: vi.fn(),
+  };
+}
+
+describe('Expanded', () => {
+  beforeEach(() => {
+    window.localStorage.setItem(
+      'amplify-sidebar-state',
+      JSON.stringify({ isOpen: true })
+    );
+  });
+  describe('Renders with correct styles', () => {
+    test('Default', () => {
+      const props = fakeProps();
+      render(<CreateItem {...props} />, {
+        wrapper: SideBarProvider,
+      });
+
+      const button = screen.getByRole('button', { name: props.createLabel });
+      const item = button.parentElement;
+      const svgPath = screen.getByTestId('eds-icon-path');
+
+      expect(item).toHaveStyleRule('display', 'flex');
+      expect(item).toHaveStyleRule('padding', '0 14px');
+      expect(item).toHaveStyleRule('align-items', 'center');
+      expect(item).toHaveStyleRule('align-self', 'stretch');
+      expect(item).toHaveStyleRule('box-sizing', 'border-box');
+      expect(item).toHaveStyleRule('height', '64px');
+      expect(item).toHaveStyleRule(
+        'border-bottom',
+        `1px solid ${colors.ui.background__medium.rgba}`
+      );
+
+      expect(button).toHaveStyleRule('width', 'fit-content');
+      expect(button).toHaveStyleRule('height', '36px');
+      expect(button).toHaveStyleRule(
+        'border-radius',
+        shape.icon_button.borderRadius
+      );
+      expect(button).toHaveStyleRule(
+        'border-radius',
+        shape.icon_button.borderRadius,
+        { modifier: ':hover' }
+      );
```

Comment:
I know you did nott write this test in this PR.
But this feels like an uncessesary amount of style checks for the test. I think it should only test for the things that change between collapsed/open etc 🤔